File: m/rls/stexc.r
file	stexc - exchange leading substring
include	rid:rider
include	rid:stdef

code	st_exc - exchange leading substring

  func	st_exc
	rep : * char			; replacement string
	str : * char~			; destination exchange pointer
	cnt : int			; exchange size
	()  : * char			; past insertion
  is	st_mov (str+cnt, str)		; squash them out
	st_ins (rep, str)		; insert remainder
	reply that			; past replacement
  end


File: m/rls/rlb/stexc.c
/* file -  stexc - exchange leading substring */
#include "m:\rid\rider.h"
#include "m:\rid\stdef.h"
/* code -  st_exc - exchange leading substring */
char *st_exc(
register char *rep ,
register char *str ,
int cnt )
{ st_mov (str+cnt, str);
  return ( (st_ins (rep, str)));
} 
